tests/conftest.py

```diff
@@ -90,7 +90,7 @@ def target_win(hive_hklm, fs_win):
     mock_target.registry.map_hive("HKEY_LOCAL_MACHINE", hive_hklm)
 
     mock_target.filesystems.add(fs_win)
-    mock_target.fs.mount("C:", fs_win)
+    mock_target.fs.mount("c:", fs_win)
```

Comment:
Why is this change necessary?

Reply:
Why is that? That sounds like a more severe bug and this is a band-aid to fix that?

Reply:
Isn't removing this line then the correct fix? The `WindowsPlugin` will mount the filesystem to the correct letter.